source/api_documentation/network_integration/ringpools/_post_ring_pools.rst

```diff
@@ -18,24 +18,13 @@
 
     {
       "id_from_network": "12",
-      "param1": "q",
-      "param2": "test",
-      "param3": "param3",
-      "param4": "param4",
-      "param5": "param5",
-      "param6": "param6",
-      "param7": "param7",
-      "param8": "param8",
-      "param9": "param9",
-      "param10": "param10",
       "local_center": { "tn_prefix":"",
                         "zipcode": "90210",
                         "latitude": 30,
                         "longitude": -130 },
       "local_number_preferences": { "search_radius_miles": 20,
                                     "restrict_to_state": true,
                                     "allocation_fallback_strategy": "Wait" },
-      "pool_type": "SearchKeyword",
       "preferred": true,
       "name": "Invoca Example RingPool",
       "destination_url": "https://www.invoca.com",
```

Comment:
are tracking url, destination url, and destination type required on create? If not can we remove them for simplicity?

Reply:
I fee like `destination_type` should be defaulted to :Advertiser in the same block that you are setting pool_type

`destination_url` looks like it can be left blank (didn't confirm locally)
I think `tracking_url` is ignored from 2013-11-20 onward, so hopefully can removed throughout docs here too?

Reply:
+1